zerver/views/home.py

```diff
@@ -215,6 +215,24 @@ def home_real(request: HttpRequest) -> HttpResponse:
         # Adding two_fa_enabled as condition saves us 3 queries when
         # 2FA is not enabled.
         two_fa_enabled_user   = two_fa_enabled and bool(default_device(user_profile)),
+        # Adding all stop words
+        stop_words            = ["i", "me", "my", "myself", "we", "our", "ours", "ourselves",
+                                 "you", "your", "yours", "yourself", "yourselves", "he", "him",
+                                 "his", "himself", "she", "her", "hers", "herself", "it", "its",
+                                 "itself", "they", "them", "their", "theirs", "themselves", "what",
+                                 "which", "who", "whom", "this", "that", "these", "those", "am",
+                                 "is", "are", "was", "were", "be", "been", "being", "have", "has",
+                                 "had", "having", "do", "does", "did", "doing", "a", "an", "the",
+                                 "and", "but", "if", "or", "because", "as", "until", "while", "of",
+                                 "at", "by", "for", "with", "about", "against", "between", "into",
+                                 "through", "during", "before", "after", "above", "below", "to",
+                                 "from", "up", "down", "in", "out", "on", "off", "over", "under",
+                                 "again", "further", "then", "once", "here", "there", "when", "where",
+                                 "why", "how", "all", "any", "both", "each", "few", "more", "most",
+                                 "other", "some", "such", "no", "nor", "not", "only", "own", "same",
+                                 "so", "than", "too", "very", "s", "t", "can", "will", "just", "don",
+                                 "should", "now", "b", "c", "d", "e", "f", "g", "h", "j", "k", "l",
+                                 "m", "n", "o", "p", "q", "r", "u", "v", "w", "x", "y", "z"],
```

Comment:
I think it makes more sense to read this from a source file on disk, rather than putting them all inline here; that's certainly easier for managing changes to the list.  Our authoritative list is `puppet/zulip/files/postgresql/zulip_english.stop`, and we can just read that path using `os.path.join(DEPLOY_ROOT, "puppet/zulip/files/postgresql/zulip_english.stop")`

Reply:
Made the following changes:
https://github.com/zulip/zulip/blob/f7137d67a70882d0c0221ec7ab2e64aad8d7aac6/zerver/views/home.py#L98-L102